### DISCUSSION TITLE
Null-safety support

### DISCUSSION THREAD

**Discussion by higorlapacw** [oddbit/flutter_facebook_app_events#88] [Ideas]
Hey guys, thanks for the awesome package! Is there any plans in migrating to null-safety? 

[here's an awesome guide in how to do it](https://dart.dev/null-safety/migration-guide).

**Comment by AlexShniperson-INO** [comment]
we also blocked by your plugin

**Reply by DennisAlund** [reply]
Could you be more specific?

**Reply by AlexShniperson-INO** [reply]
we use your plugin in our project and because it's not null safe yet we can't upgrade to Flutter 2.0

**Reply by DennisAlund** [reply]
Gotcha... there's a PR for it (#90). But I haven't had time to look into upgrading to `2.0` myself yet.

**Reply by DennisAlund** [reply]
It's been resolved in PR #90

**Comment by DennisAlund** [comment]
Thanks for the input @higorlapacw time is an issue for us, so we'd highly appreciate and welcome a pull request from you :-)

**Reply by DennisAlund** [reply]
It's been resolved in PR #90
